### DISCUSSION TITLE
This MongoDB deployment does not support retryable writes

### DISCUSSION THREAD

**Discussion by vimalmistry** [prisma/prisma#8701] [Q&A]
Hi, I installed mongodb-community edition on my mac. and Initialized with prisma.

I installed version 5.0.1

```prisma
datasource db {
  provider = "mongodb"
  url      = "mongodb://localhost:27017/prisma_mongo?retryWrites=false&w=majority"
}

generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["mongoDb"]
}

model User {
  id        Bytes    @id @default(dbgenerated()) @map("_id") @db.ObjectId
  // Other fields
  name      String
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
}

```

Here is my code.
```js
import x from "@prisma/client";
const prisma = new x.PrismaClient();

const res = await prisma.user.create({
  data: { name: "cyz" },
});

console.log({ res });

```

I am getting this error
```
ConnectorError(ConnectorError { user_facing_error: None, kind: RawError { code: "unknown", message: "Command failed (IllegalOperation): This MongoDB deployment does not support retryable writes. Please add retryWrites=false to your connection string.)
```
As you can see I changed database url also. Bt still facing this problem. Please do not suggest docker.  I want to know what's the problem. Because I have to setup same environment in my vps also. without docker :)

**Comment by ryands17** [comment]
@vimalmistry 👋 

Here's the response to a similar question that you have: https://stackoverflow.com/questions/58589631/mongoerror-this-mongodb-deployment-does-not-support-retryable-writes-please-ad

Let me know if that was what you wanted 🙂

**Reply by Tchoupinax** [reply]
Hello,

I meet the same issue. Is it a Prisma issue ?

**Reply by ryands17** [reply]
As suggested in the above StackOverflow answer, you need to use a replica set to run this. The solution specified in the answer will work for you.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
